src/main/java/org/opensearch/knn/plugin/script/KNNScoreScript.java

```diff
@@ -139,4 +140,6 @@ public double execute(ScoreScript.ExplanationHolder explanationHolder) {
             return this.scoringMethod.apply(this.queryValue, scriptDocValues.getValue());
         }
     }
+
+    public static final ScriptContext<KNNScoreScriptFactory> CONTEXT = new ScriptContext<>("knn_score", KNNScoreScriptFactory.class);
```

Comment:
could you please make "knn_score" a constant

Reply:
Actually, after re-looking at this, the code clearly shows it uses the context of its superclass so I reverted this code entirely.